alpha/alpha.ino

```diff
@@ -8,95 +8,168 @@
 
 bool powerSaveTest = false;
 
+bool SavePower = false;
+int stackSize = 1000;
+unsigned long debounceDelay = 50; // the debounce time; increase if the output flickers
+
 void setup() {
   TS_HAL.begin();
   TS_HAL.ble_init();
-  
+
   // Reduce screen brightness to minimum visibility to reduce power consumption
   TS_HAL.lcd_brightness(12);
 
   TS_HAL.lcd_cursor(40, 0);
   TS_HAL.lcd_printf("ALPHA TEST");
 
-  if(powerSaveTest)
+  if (powerSaveTest)
   {
     TS_HAL.lcd_backlight(false);
     TS_HAL.lcd_sleep(true);
   }
 
   OT_ProtocolV2.begin();
-  
+
+  xTaskCreate(
```

Comment:
do Trace things in loop().
this would also control when the system goes into light sleep.
we'll need a a flag so that secondary tasks can prevent the system from going into light sleep, e.g. UI/serial/charging is activated.

Reply:
I'm worried that doing trace things in Loop would mean that it'll default to be the lowest priority task